src/librustdoc/lib.rs

```diff
@@ -462,6 +462,14 @@ fn opts() -> Vec<RustcOptGroup> {
                 "human|json|short",
             )
         }),
+        stable("diagnostic-width", |o| {
```

Comment:
@rust-lang/rustdoc this would insta-stable for rustdoc, would you prefer I add it as unstable first?